inc/taginjection.class.php

```diff
@@ -28,57 +28,66 @@
  * -------------------------------------------------------------------------
  */
 
-class PluginTagTagInjection extends PluginTagTag
-      implements PluginDatainjectionInjectionInterface {
+/** @phpstan-ignore-next-line */
```

Comment:
Ignore does not work, error `Class PluginTagTagInjection implements unknown interface PluginDatainjectionInjectionInterface.` persists :(

Reply:
A solution would be to add the interface in a `stubs/PluginDatainjectionInjectionInterface.php` file and add the following entry in the `phpstan.neon` file:
```yaml
    stubFiles:
        - stubs/PluginDatainjectionInjectionInterface.php
```

Reply:
I've just tried, that does not work

Reply:
I wonder if the best solution would not be to adddatainjection plugin